isvcs/zk.go

```diff
@@ -78,31 +78,46 @@ func initZK() {
 
 // a health check for zookeeper
 func zkHealthCheck(halt <-chan struct{}) error {
-	lastError := time.Now()
+	// minUptime til zookeeper is considered available
 	minUptime := time.Second * 2
-	zookeepers := []string{"127.0.0.1:2181"}
-
+	// timer to verify that the service has been available for the min duration
+	timer := time.NewTimer(minUptime)
+	timer.Stop()
```

Comment:
because we don't want the timer running while we are in the process of acquiring a connection